src/utils/hooks/useUpdateReviewResponseNEW.ts

```diff
@@ -0,0 +1,57 @@
+import {
+  ReviewResponse,
+  ReviewResponseDecision,
+  ReviewResponseRecommendedApplicantVisibility,
+  useUpdateReviewResponseMutation,
+} from '../generated/graphql'
+
+// below lines are used to get return type of the function that is returned by useUpdateReviewResponseMutation
+type UseUpdateReviewMutationReturnType = ReturnType<typeof useUpdateReviewResponseMutation>
+type promiseReturnType = ReturnType<UseUpdateReviewMutationReturnType[0]>
```

Comment:
Should we keep consistency with the standard that types and interfaces are written with capitals? So: `PromiseReturnType`?

Reply:
indeed !
bca100f36430f3126044b34e9d2b4676f5b83bdb